soc/arc/snps_qemu/Kconfig

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2020 Synopsys, Inc. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+choice
+	prompt "ARC QEMU SoC Selection"
+	default SOC_QEMU_ARC_HS
+	depends on SOC_QEMU_ARC
+
+config SOC_QEMU_ARC_EM
+	bool "Synopsys ARC EM in nSIM"
```

Comment:
typo / copy&paste mistake: nSIM -> QEMU (also below for HS)

Reply:
Copypasta :(
Fixed, thanks.